src/components/ArticleCard.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { Typography, Card, CardContent, CardActionArea } from "@mui/material";
+
+const ArticleCard = ({ article }) => {
+  return (
+    <Card data-cy="current-user-articles" sx={{ width: "100%", maxWidth: 700 }}>
+      <CardActionArea>
```

Comment:
just after this line, you may add the image if present:
`        {article.image && (
          <CardMedia component="img" height="140" image={article.image} />
        )}`